public/index.ejs

```diff
@@ -271,6 +344,17 @@ function updateFollowingUI(users) {
     }
   }
 }
+
+function abortPendingFetches() {
+  // Abort all fetches since last UI update.
+  abortControllersList.map(item => {
```

Comment:
Does this need to be map()?

Reply:
It can be an iterator, I don't have a preference.  We were using map elsewhere, so I changed it for consistency.

Reply:
I see. I think the only other place I'm seeing `map` is when we actually need the array returned from it. I'll go ahead and make this a forEach.